<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Punkte</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="whatsThis">
       <string/>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabEingabe">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QTableView" name="tableView"/>
        </item>
        <item row="0" column="0">
         <widget class="QListView" name="listView">
          <property name="viewMode">
           <enum>QListView::ListMode</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushNKurs">
          <property name="text">
           <string>Neuer Kurs</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushNBlatt">
          <property name="text">
           <string>Neues Blatt</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>508</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionNeu"/>
    <addaction name="action_ffnen"/>
    <addaction name="action_Speichern"/>
    <addaction name="actionSpeichern_unter"/>
    <addaction name="separator"/>
    <addaction name="action_Beenden"/>
   </widget>
   <widget class="QMenu" name="menuBearbeiten">
    <property name="title">
     <string>Bearbeiten</string>
    </property>
    <addaction name="actionNeuer_Kurs"/>
    <addaction name="actionNeues_Blatt"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuBearbeiten"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNeu">
   <property name="text">
    <string>&amp;Neu</string>
   </property>
  </action>
  <action name="action_ffnen">
   <property name="text">
    <string>&amp;Ã–ffnen</string>
   </property>
  </action>
  <action name="action_Speichern">
   <property name="text">
    <string>&amp;Speichern</string>
   </property>
  </action>
  <action name="actionSpeichern_unter">
   <property name="text">
    <string>Speichern &amp;unter</string>
   </property>
  </action>
  <action name="action_Beenden">
   <property name="text">
    <string>&amp;Beenden</string>
   </property>
  </action>
  <action name="actionNeuer_Kurs">
   <property name="text">
    <string>Neuer Kurs</string>
   </property>
  </action>
  <action name="actionNeues_Blatt">
   <property name="text">
    <string>Neues Blatt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>listView</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>pushNKurs</tabstop>
  <tabstop>pushNBlatt</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
